### ISSUE TITLE
Error starting neo4j-client unless --no-history flag is used

### ISSUE THREAD

**Issue by bavalpey** [cleishm/libneo4j-client#41]
Hello,

I have been running into troubles starting neo4j-client.  Unless the '--no-history' flag is set, attempting to start neo4j-client results in the following:
Failed to load history: Success

I am using Debian.

Not sure what other logs to include..

**Comment by bavalpey** [comment]
Hm.. I fixed it after deleting the history file.  If anyone else has this happen, find where the history file is (for me it was ~/.neo4j/client-history) and delete it.

**Comment by cleishm** [comment]
Hi @bavalpey!

You don't happen to have the old history file still copied somewhere? I'd love to see why it failed to load and update that error to be more helpful :)

Cheers,
Chris

**Comment by bavalpey** [comment]
@cleishm 
Ah, no I don't have the old history file copied somewhere.

However, before deleting it, I did open it to see what it was, and it was exactly one line with nothing written on it (there may have been whitespace).

It's probably important to note that the first time I started neo4j-client, I suspended the process with CTRL+Z and then terminated the process (I did not exit using :quit). Since then, I had received the 'FAILED TO LOAD HISTORY: SUCCESS' error.  I searched through this repository to find references to that error, and after finding what the default name for the history file was, searched for that file using find (I was unaware that the files were kept in ~/.neo4j).

Hope that helps, I am sure it is related to me force-quitting the process.
